eva-accession-release/src/main/resources/create_release_properties.sh

```diff
@@ -0,0 +1,104 @@
+	mongo_port=`grep -m 1 "^spring.data.mongodb.port" $IMPORT_FILE | sed "s/^[^=]\+=//"`
+	mongo_db=`grep -m 1 "^spring.data.mongodb.database" $IMPORT_FILE | sed "s/^[^=]\+=//"`
+	mongo_user=`grep -m 1 "^spring.data.mongodb.username" $IMPORT_FILE | sed "s/^[^=]\+=//"`
+	mongo_pass=`grep -m 1 "^spring.data.mongodb.password" $IMPORT_FILE | sed "s/^[^=]\+=//"`
+	mongo_authdb=`grep -m 1 "^spring.data.mongodb.authentication-database" $IMPORT_FILE | sed "s/^[^=]\+=//"`
+
+	OUTPUT_FOLDER=${OUTPUT_RELEASE_FOLDER}/${assemblyAccession}/
+
+	echo ""
+	echo "
+spring.batch.job.names=ACCESSION_RELEASE_JOB
+
+parameters.assemblyAccession=${assemblyAccession}
+parameters.assemblyReportUrl=${assemblyReport}
+parameters.chunkSize=1000
+parameters.fasta=${fasta}
+parameters.forceRestart=false
+parameters.outputFolder=${OUTPUT_FOLDER}
+
+# job repository datasource
+spring.datasource.driver-class-name=org.postgresql.Driver
+spring.datasource.url=${db_url}
+spring.datasource.username=${db_user}
+spring.datasource.password=${db_pass}
+spring.datasource.tomcat.max-active=3
+
+# Only to set up the database!
+# spring.jpa.generate-ddl=true
+
+# MongoDB for storing imported accessions
+spring.data.mongodb.host=${mongo_host}
+spring.data.mongodb.port=${mongo_port}
+spring.data.mongodb.database=${mongo_db}
+spring.data.mongodb.username=${mongo_user}
+spring.data.mongodb.password=${mongo_pass}
+spring.data.mongodb.authentication-database=${mongo_authdb}
+mongodb.read-preference=primaryPreferred
+
+spring.main.web-environment=false
+
+logging.level.uk.ac.ebi.eva.accession.release=INFO
+" > ${OUTPUT_FILE}
+
+	echo "# you will likely want to run these jobs (CHECK THAT THE VERSION OF THE JAR FILE IS UPDATED!! https://github.com/EBIvariation/eva-accession/tags):"
+	echo "
+mkdir -p ${OUTPUT_FOLDER}
+jobname=\`date +\"%Y%m%d_%H%M%S\"\`_${assemblyAccession}
+bsub -J \${jobname} -o release.log -e release.err -M 8192 -R \"rusage[mem=8192]\" \"java -Xmx7g -jar ${EVA_ACCESSION_JAR_PATH}\"
+bsub -J \${jobname}_sort -w \${jobname} -o sort_ids.log -e sort_ids.err -M 8192 -R \"rusage[mem=8192]\" \"sort -V ${OUTPUT_FOLDER}${assemblyAccession}_deprecated_ids.unsorted.txt | uniq | gzip > ${OUTPUT_FOLDER}${assemblyAccession}_deprecated_ids.txt.gz; sort -V ${OUTPUT_FOLDER}${assemblyAccession}_merged_deprecated_ids.unsorted.txt | uniq | gzip > ${OUTPUT_FOLDER}${assemblyAccession}_merged_deprecated_ids.txt.gz\"
+bsub -J \${jobname}_validate -w \${jobname} -o ${OUTPUT_FOLDER}validate.log -e ${OUTPUT_FOLDER}validate.err \"${VCF_VALIDATOR_PATH} -i ${OUTPUT_FOLDER}${assemblyAccession}_current_ids.vcf -o ${OUTPUT_FOLDER}; ${VCF_VALIDATOR_PATH} -i ${OUTPUT_FOLDER}${assemblyAccession}_merged_ids.vcf -o ${OUTPUT_FOLDER}; ${VCF_ASM_CHECKER_PATH} -i ${OUTPUT_FOLDER}${assemblyAccession}_current_ids.vcf -f ${fasta} -o ${OUTPUT_FOLDER} -r text,summary ; ${VCF_ASM_CHECKER_PATH} -i ${OUTPUT_FOLDER}${assemblyAccession}_merged_ids.vcf -f ${fasta} -o ${OUTPUT_FOLDER} -r text,summary ;\"
```

Comment:
We are not waiting for the sort to finish. So, if the validation fails and we need to regenerate the VCF for some reason, won't the sort go to waste (may be this is rare and we don't need to worry about this)? Also, do we get a lot of value running these processes in parallel?  If there are intermediate failures, this might aggravate the number of stale jobs that are spawned and might make tracking difficult...

Reply:
Actually, there's another problem. I only made the compression depend on the validation so that it waits to delete the original VCF. We will have to compress and release some VCFs that won't pass the validator, like having duplicates.
But now I realise that if the validation job fails, the others won't start. I have to change this so that they start if the validation job is finished or failed.